app/views/core_induction_programmes/years/_module_list.html.erb

```diff
@@ -0,0 +1,22 @@
+<% if course_modules.each do |course_module| %>
+  <ul class="app-task-list__items">
+    <li>
+      <div class="app-task-list__item">
+        <h3 class="govuk-heading-m">
+          <%= govuk_link_to(course_module.title, module_path(course_module)) %>
+        </h3>
+
+        <div class="gem-c-govspeak govuk-govspeak">
+          <%= course_module.ect_summary_to_html.html_safe %>
+        </div>
+
+        <div class="app-task-list__section">
+          <span><%= course_module.course_lessons.count.humanize.capitalize %> self-study sessions</span>
+          <%= render ProgressLabelComponent.new progress: course_module.progress %>
+        </div>
```

Comment:
I would ask if we want to pluralise `sessions` but I don't think there is any module with a single session, sooo... we can probably let it slide.